php-resource/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "sample-php-resource",
+    "type": "module",
+    "description": "Sample localization project that localizes strings into php resource files.",
+    "repository": "git@github.com:iLib-js/ilib-loctool-samples.git",
+    "license": "Apache-2.0",
+    "version": "1.0.0",
+    "scripts": {
+        "loc": "loctool",
+        "debug": "node --inspect-brk node_modules/loctool/loctool.js",
+        "clean": "rm -rf *.xliff resources",
+        "test": "echo success"
+    },
+    "devDependencies": {
+        "ilib-loctool-javascript": "^1.1.3",
+        "loctool": "^2.25.2"
+    },
+    "dependencies": {
```

Comment:
Actually, why put in any prod dependencies at all? Assuming this `php-resource/package.json` is meant to be as if someone created a new package and wanted to use `loctool` and `ilib-loctool-php-resource` to extract strings, both of those packages should be devdependencies of this package